scripts/travis/travis_build_and_test.sh

```diff
@@ -6,7 +6,7 @@ set -e
 MAKE="make --jobs=$NUM_THREADS --keep-going"
 
 if $WITH_CMAKE; then
-  mkdir build
+  mkdir -p build
```

Comment:
Yes that prevents it from failng when build directory exists - why shouldn't we check it in, it won't break anything.